### DISCUSSION TITLE
Configuration - group_folder_column:

### DISCUSSION THREAD

**Discussion by mjh8431** [RafaelGB/obsidian-db-folder#302] [Q&A] [answered]
In reading through the documentation (which has already been admitted as sketchy) I found this entry, which piqued my interest. There is nothing in the set up that allows this to be set. The only thing I found was in looking at the actual DB-Folder markdown file: 
config:
  remove_field_when_delete_column: "true"
  cell_size: "wide"
  sticky_first_column: "false"
  group_folder_column: ""

Is this where you set it? And what does it actually do? I know the documentation says it groups like identified files into subfolders, but it would be nice to have a little further explanation of the resulting action since I am not a programmer.

Hopefully, there will be a more robust documentation for the plugin coming soon.

Thanks

**Comment by RafaelGB** [comment]
Hi @mjh8431
yes, the documentation has rom for improvement ([any help is welcome](https://rafaelgb.github.io/obsidian-db-folder/contributing/documentationContribution/))

To group notes into subfolders you need a select column type first, then you can use that field as group.


https://user-images.githubusercontent.com/11924043/187606429-7b98850f-1f6c-4c73-9337-303b0edb0d00.mov

**Reply by mjh8431** [reply]
Thanks for the reply. It took me a bit to remember where the database setting were located.

I tried it. Here is what I found. Once the column for the group folder is set, I have to make sure the column property type = Select. Then I need to create appropriate label options in the column settings (or they will be created automatically if I had created them when the column property = Text). When I choose the column label option for a file in the database, a sub folder is created (if not previously created), the file is moved to that sub folder, and is removed from the database file. For all files in the database that I created previously (over 170) I have to manually select the label option.
So, what if I want to keep it in the database file and maintain the group folder?  The two options I see is to create a query that includes the subfolders - I am new to this so I don't know if that is possible - or add a tag to the metadata of each file that is the same as the label used in the column for the group folder. Adding the tag would also have to be done manually for each file in the database.
Am I missing something?

**Comment by RafaelGB** [comment, marked as answer]
your request sounds more like a formula column or the related column type. they are future issues to add. You can fins them into the roadmap > long term.

Actually you meed to add manually that select into all your files I am afraid
